File: openmp-wg-pII-2015/omp_section.f90
program omp_section
    use omp_lib

!$OMP PARALLEL SECTIONS
!$OMP SECTION
    write(*,"(A,I4)") "Hello 1 from thread ",omp_get_thread_num()

!$OMP SECTION
    write(*,"(A,I4)") "Hello 2 from thread ",omp_get_thread_num()

!$OMP END PARALLEL SECTIONS

end program

File: openmp-wg-pII-2015/omp_section.c
#include <stdio.h>

int main(){
    #pragma omp parallel sections
    {
        #pragma omp section
            printf("Hello 1 from thread %d\n",omp_get_thread_num());
        #pragma omp section
            printf("Hello 2 from thread %d\n",omp_get_thread_num());
    }
}
